Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/ComponentEditor.cpp

```diff
@@ -214,8 +211,34 @@ namespace AzToolsFramework
             if (!aggregateInstance)
             {
                 // Set the adapter component to this instance.
-                // Note: multiple selection with DPE is not yet supported
                 m_adapter->SetComponent(componentInstance);
+
+                // set the DPE back to the regular adapter and destroy the aggregate adapter
+                if (m_aggregateAdapter)
+                {
+                    m_dpe->SetAdapter(m_adapter);
```

Comment:
Should this be set back to m_filterAdapter?

Reply:
wow, good catch! Yep, totally. Fixed now, thanks!